src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -716,5 +719,21 @@ public void ResetLoadingStatusIndicator()
         {
             _loadingStatusIndicator.Reset(string.Empty);
         }
+
+        internal void ClearPackageLevelTypeGrouping()
+        {
+            ItemsView.GroupDescriptions.Clear();
+        }
+
+        internal void AddPackageLevelTypeGrouping()
+        {
+            ItemsView.Refresh();
```

Comment:
I see this is a heavier refresh, but I think it's fine since it will only occur when first switching to a tab that supports grouping.

Reply:
As noted, this only occurs when needed to check if there are any transitive dependencies in the list before adding grouping. Resolving comment.